frontend/tool/webpack.demo.config.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const webpack = require('webpack');
+const FriendlyErrorsPlugin = require('friendly-errors-webpack-plugin');
+let merge = require('webpack-merge');
+let baseWebpackConfig = require('./webpack.config');
+
+// add hot-reload related code to entry chunks
+Object.keys(baseWebpackConfig.entry).forEach(function (name) {
+    baseWebpackConfig.entry[name] = ['./tool/dev-client'].concat(baseWebpackConfig.entry[name]);
```

Comment:
Not sure if we need to update the name from `./tool/dev-client` to ./tool/demo-client